src/main/java/com/igeekinc/util/logging/iGeekLogger.java

```diff
@@ -13,135 +13,6 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
- 
```

Comment:
If the class definition isn't used by any others, how about deleting the entire file?

Reply:
Agreed